src/EventHandler.cpp

```diff
@@ -13,7 +13,7 @@
 
 // UGLY ASS HACK!
 // This is required for mouse grabbing in the EventHandler class.
-extern SDL_Window* _WINDOW;
+extern SDL_Window* _underlyingWindow;
```

Comment:
How come this one was renamed, but retains a leading underscore?

The variable looks to be defined at top level scope, so may still be reserved.